sdk/src/main/java/com/karhoo/sdk/api/datastore/user/UserStore.kt

```diff
@@ -1,11 +1,14 @@
 package com.karhoo.sdk.api.datastore.user
 
+import com.karhoo.sdk.api.model.Provider
 import com.karhoo.sdk.api.model.UserInfo
 
 interface UserStore {
 
     val currentUser: UserInfo
 
+    var paymentProvider: Provider?
```

Comment:
could we rename `Provider` to `PaymentProvider` ? :) other than that all lgtm!

Reply:
It's a bit of weird thing with the response model. There's a provider within a provider so the main response is called PaymentProvider and the child is called Provider. Guess I could save the whole model? 🤔